clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java

```diff
@@ -72,6 +73,7 @@ public Options options() {
     options.addOption(createArgOption("l", USER, "user name"));
     options.addOption(createArgOption("g", GROUP, "group name"));
     options.addOption(createArgOption("t", TAG, "tag name"));
+    options.addOption(createSimpleOption("o", OWNER, "display entity owner"));
```

Comment:
What is "display entity owner"?

Reply:
It displays an entities owner

Reply:
"entity owner" is enough, where "display" is from?

Reply:
Done